src/shared/components/Select/Select.stories.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react'
+import Select, { SelectProps, SelectedItem } from '.'
+import { Story, Meta } from '@storybook/react'
+
+const items: SelectedItem[] = [
+  { name: 'first', value: 'first' },
+  { name: 'second', value: 'second' },
+  { name: 'third', value: 'third' },
+]
+
+export default {
+  title: 'Shared/Select',
+  component: Select,
+  args: {
+    items,
+    label: 'change value',
+    placeholder: 'placeholder',
+    disabled: false,
+    error: '',
+    helperText: '',
+    warning: '',
+  },
+} as Meta
+
+const Template: Story<SelectProps> = (args) => <Select {...args} />
+
+const TemplateWithControlledInput: Story<SelectProps> = (args) => {
+  const [selectedItem, setSelectedItem] = useState<SelectedItem>({
+    value: 'selected value',
+    name: 'selected value',
+  })
+  return (
+    <Select
+      {...args}
+      onChange={({ selectedItem }) => selectedItem && setSelectedItem(selectedItem)}
+      value={selectedItem}
+    />
+  )
+}
+
+export const Default = Template.bind({})
+
+export const WithControlledInput = TemplateWithControlledInput.bind({})
+
+export const Disabled = Template.bind({})
```

Comment:
I don't think separate Disabled, Error and Warning is needed since we can achieve that via controls on the Default.